ruby/game/game.rb

```diff
@@ -0,0 +1,234 @@
+# ----- PSEUDOCODE
+# Input word
+  # Loop until guess_count > word.length + 3
+    # Display word as spaces + guessed char (guessed_word)
+    # Input guess (char)
+    # Compare guess to word
+    # Check if char already guessed
+        # If so, prompt again
+    # Increment guess_count by 1
+    # Loop through array of word char's
+      # If guess = char in word, modify guessed_word at index
+        # Print correct guess
+      # Else
+        # Print incorrect guess
+    # Break if no '_' left in guessed_word array
+      # Declare 
+# Print either win or lose
+  # Based on won_game boolean
+
+class Game
+  attr_accessor :word, :guess_count, :guess, :guessed_word, :guessed_letters, :is_over, :won_game, :already_guessed
+  
+  def initialize 
+    @word = []
+    @guess = ""
```

Comment:
Do you need to save guess as an attribute since it supposed to change every time there is a new guess?